users/views/admin/settings.py

```diff
@@ -37,6 +37,10 @@ class BasicSettings(AdminSettingsPage):
             "title": "Maximum Post Length",
             "help_text": "The maximum number of characters allowed per post",
         },
+        "content_warning_text": {
+            "title": "Content Warning Placeholder Text",
```

Comment:
Let's call it "Content Warning Feature Name", and the help_text should be "What the feature that lets users provide post summaries is called"